### PR TITLE
Added some usage-examples

### PR DESCRIPTION
**Pull request by JlnWntr**

### CODE REVIEW

File: programs/http_client.c [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 /*
  * Usage: http_client remote_addr remote_port [local_port] [local_encaps_port] [remote_encaps_port] [uri]
+ * 
+ * Example?
```
Comment (tuexen) [inline programs/http_client.c]: Remove this or provide an example...

Comment (JlnWntr) [reply programs/http_client.c]: Yes, sir! ...

**Comment by tuexen** [comment]
Thanks for the patch!

Best regards
Michael

**Comment by lgrahl** [comment]
Thanks for making my day to both of you. :grin:

**Comment by JlnWntr** [comment]
What's so funny Lennart? You're welcome! :)

**Pull request merged**